halo/registry/types/interface.go

```diff
@@ -10,6 +10,6 @@ type PortalRegistry interface {
 	// SupportedChain returns true if the chain is included in the registry.
 	SupportedChain(ctx context.Context, chainID uint64) (bool, error)
```

Comment:
Why don't we clean this up while we are here:
 - Remove `SupportedChain` from interface, implementation and tests
 - Rename `ConfLevels` to something like `SupportedChainVersions`
 - Refactor all uses of `SupportedChain` to use `SupportedChainVersions`.
 - Ensure we have unit tests for `SupportedChainVersions`.